features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the previous and current branches are deleted during a Git Town command, the main branch becomes the new previous branch
```

Comment:
I think the distinction between direct effect and side effect is semantic here. Its an effect, and the one that this spec describes. So let's go with the one that is more concise and to the point, which is mine here. Thanks!  :)